src/utils/deploy.ts

```diff
@@ -180,7 +184,9 @@ export async function executeDeploy(
   }
 
   // does not apply to mdapi deploys
-  const manifestPath = componentSet ? await writeManifest(deploy.id, componentSet) : undefined;
+  const manifestPath = componentSet
+    ? await writeManifest(deploy.id, componentSet, registry ?? new RegistryAccess())
```

Comment:
passed in the registry here to make sure the manifest knows about customRegistries